root/etc/e-smith/events/actions/nethserver-directory-tlspolicy

```diff
@@ -20,14 +20,18 @@
 # along with NethServer.  If not, see COPYING.
 #
 
+#
+# Following REHL recommendation https://access.redhat.com/articles/1474813
+#
+
 tlspolicy=$(/sbin/e-smith/config getprop tls policy)
 
 if (( ${tlspolicy:-0} >= 20200510 )); then
     ldapmodify -c -Y EXTERNAL &>/dev/null <<'EOF' || ((errors ++))
 dn: cn=config
 changetype: modify
 replace: olcTLSCipherSuite
-olcTLSCipherSuite: EECDH:EDH:CAMELLIA:ECDH:RSA:!eNULL:!SSLv2:!RC4:!DES:!EXP:!SEED:!IDEA:!3DES
```

Comment:
:+1: From https://gist.github.com/stephdl/984fdf81a9dd177bb7c7723833930d22 it seems this change just adds some new ciphers to those already available. In this case we can just fix the latest TLS policy.

Let me check that diff by my side too during the QA ;)